src/ray/core_worker/task_manager.cc

```diff
@@ -28,6 +28,7 @@
 #include "ray/gcs/pb_util.h"
 #include "ray/util/exponential_backoff.h"
 #include "ray/util/util.h"
+#include "src/ray/protobuf/common.pb.h"
```

Comment:
```suggestion
#include "ray/protobuf/common.pb.h"
```
having the src there causes issues on windows builds